### PR TITLE
linuxPackages.oci-seccomp-bpf-hook: 1.2.0 -> 1.2.1

### PR DESCRIPTION
**Pull request by saschagrunert**
###### Motivation for this change
Update to the latest release.

cc @NixOS/podman 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/linux/oci-seccomp-bpf-hook/default.nix [hunk 1]
```diff
@@ -10,12 +10,12 @@
...
+  version = "1.2.1";
   src = fetchFromGitHub {
     owner = "containers";
     repo = "oci-seccomp-bpf-hook";
     rev = "v${version}";
-    sha256 = "143x4daixzhhhpli1l14r7dr7dn3q42w8dddr16jzhhwighsirqw";
+    sha256 = "0zbrpv6j4gd4l36zl2dljazdm85qlqwchf0xvmnaywcj8c8b49xw";
   };
   vendorSha256 = null;
   doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/oci-seccomp-bpf-hook/default.nix:21]: Can you add a comment why no tests are run?

Comment (saschagrunert) [reply pkgs/os-specific/linux/oci-seccomp-bpf-hook/default.nix]: Hm, let me check if we can get those tests working…

File: pkgs/os-specific/linux/oci-seccomp-bpf-hook/default.nix [hunk 2]
```diff
@@ -10,15 +10,14 @@
...
+  version = "1.2.1";
   src = fetchFromGitHub {
     owner = "containers";
     repo = "oci-seccomp-bpf-hook";
     rev = "v${version}";
-    sha256 = "143x4daixzhhhpli1l14r7dr7dn3q42w8dddr16jzhhwighsirqw";
+    sha256 = "0zbrpv6j4gd4l36zl2dljazdm85qlqwchf0xvmnaywcj8c8b49xw";
   };
   vendorSha256 = null;
-  doCheck = false;
```
Comment (zowoq) [inline pkgs/os-specific/linux/oci-seccomp-bpf-hook/default.nix:21]: Removing this doesn't run the tests as the default `{build,install}Phase` is overridden, would need to run `make check` or similar in `checkPhase`.

**Comment by zowoq** [comment]
Commit message should be `linuxPackages.oci-seccomp-bpf-hook: 1.2.0 -> 1.2.1`

File: pkgs/os-specific/linux/oci-seccomp-bpf-hook/default.nix (hunk 2 above)
Comment (saschagrunert) [reply pkgs/os-specific/linux/oci-seccomp-bpf-hook/default.nix:21]: Alright, fixed by adding a `go test -v ./...`

**Comment by saschagrunert** [comment]
> Commit message should be `linuxPackages.oci-seccomp-bpf-hook: 1.2.0 -> 1.2.1`

Fixed as suggested :+1:

**Review by vdemeester — APPROVED** [review]
LGTM 🐯

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110931` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>linuxPackages-libre.oci-seccomp-bpf-hook: log was empty</li>
  </ul>
</details>
<details>
<summary>
12 packages built:
</summary>
<ul>
<li>linuxPackages.oci-seccomp-bpf-hook</li>
<li>linuxPackages_5_10.oci-seccomp-bpf-hook</li>
<li>linuxPackages_hardened.oci-seccomp-bpf-hook</li>
<li>linuxPackages_latest-libre.oci-seccomp-bpf-hook</li>
<li>linuxPackages_latest_hardened.oci-seccomp-bpf-hook</li>
<li>linuxPackages_latest_xen_dom0.oci-seccomp-bpf-hook</li>
<li>linuxPackages_latest_xen_dom0_hardened.oci-seccomp-bpf-hook</li>
<li>linuxPackages_lqx.oci-seccomp-bpf-hook</li>
<li>linuxPackages_testing_bcachefs.oci-seccomp-bpf-hook</li>
<li>linuxPackages_xen_dom0.oci-seccomp-bpf-hook</li>
<li>linuxPackages_xen_dom0_hardened.oci-seccomp-bpf-hook</li>
<li>linuxPackages_zen.oci-seccomp-bpf-hook</li>
</ul>
</details>

**Pull request merged**